dashboard/src/locales/messages/index.ts

```diff
@@ -158,7 +158,7 @@ export const messages = {
     'logSheet.fileSize': 'File Size',
     'logSheet.indexOf': 'Index of {link}',
     'logSheet.title': 'Logs Viewer',
-    'routes.deviceMonitor': 'Devices',
+    'routes.hardware': 'Hardware',
```

Comment:
I think that keeping the "Monitor" suffix would be better to keep consistency, it identifies it as being a route, just like "treeMonitor"

Reply:
done